src/improvements/tasks/eth/004-opcm-upgrade-v300-base/.env

```
@@ -0,0 +1 @@
+TENDERLY_GAS=1060405396
```

That's a big number. This should cover it.

```suggestion
TENDERLY_GAS=20000000
```